salt/_states/metalk8s_kubernetes.py

```diff
@@ -158,6 +159,25 @@ def object_present(name, manifest=None, **kwargs):
     new = __salt__["metalk8s_kubernetes.replace_object"](
         name=name_arg, manifest=manifest, old_object=obj, saltenv=__env__, **kwargs
     )
+    # Handle the conflict error by getting the most up to date object
+    # and retrying
+    if new == "409":
+        for _ in range(5):
```

Comment:
I would prefer to have this loop on the whole state, because let's imagine the objects get deleted and not just updated (I think there is no real case like that, but I still prefer to run the whole state logic on each loop)